src/Sylius/Bundle/CoreBundle/Controller/ProductTaxonController.php

```diff
@@ -44,7 +44,7 @@ public function updatePositionsAction(Request $request): Response
             /** @psalm-var array{position: string|int, id: int} $productTaxon */
             foreach ($productTaxons as $productTaxon) {
                 try {
-                    $this->updatePositions($productTaxon['position'], $productTaxon['id']);
+                    $this->updatePositions((string) $productTaxon['position'], $productTaxon['id']);
```

Comment:
There is a second `$this->updatePositions` call at this file, which should be covered as well